composer.json

```diff
@@ -408,6 +408,7 @@
             "npm run stylelint-themes"
         ],
         "va:test:accessibility": [
+            "apt-get install chromium",
```

Comment:
We will need to do this at the OS level stages for Lando, Tugboat and BRD (yum).